falcon/request.py

```diff
@@ -514,6 +516,35 @@ def cookies(self):
 
         return self._cookies.copy()
 
+    @property
+    def access_route(self):
+        """If a forwarded header exists this is a list of all ip addresses
+        from the client ip to the last proxy server.
+
+        Inspired by werkzeug's access_route
+        """
+        if self._cached_access_route is None:
+            access_route = []
+            if 'HTTP_FORWARDED' in self.env:
+                access_route = self._parse_rfc_forwarded()
+            if not access_route and 'HTTP_X_FORWARDED_FOR' in self.env:
+                # we don't handle X-Fowarded-By since it is not possible to
```

Comment:
Can this comment be removed, since access_route is only concerned with `for` addresses now?